src/sandbox.rkt

```diff
@@ -307,16 +312,17 @@
              (representation-name repr)
              (map (curry map representation-name) (test-conversions test))
              (test-vars test)
-             (prog->fpcore (test-input test) repr) #f
+             (prog->fpcore (test-input test) repr) 
+             #f
              (prog->fpcore (test-spec test) repr)
-             (and (test-output test) (prog->fpcore (test-output test) repr))
+             (test-output test)
```

Comment:
I think we still need this? Because what about timeouts or crashes, which don't have an output?

Reply:
I don't think we need this anymore because `(test-output test)` is now a dictionary and not a singular function, where all keys are expressions. Also all the keys are already in the fpcore form and not prog, so we don't need this conversion anymore I think.